x-pack/legacy/plugins/siem/public/hooks/api/helpers.test.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getIndexPatternTitleIdMapping, getIndexPatternTitles } from './helpers';
+import { mockIndexPatternSavedObjects } from './__mock__/api';
+
+describe('helpers', () => {
+  describe('getIndexPatternTitles', () => {
+    test('returns empty array when no index patterns are provided', () => {
+      const indexPatternTitles = getIndexPatternTitles([]);
+      expect(indexPatternTitles.length).toEqual(0);
+    });
+
+    test('returns titles when index patterns are provided', () => {
+      const indexPatternTitles = getIndexPatternTitles(mockIndexPatternSavedObjects);
+      expect(indexPatternTitles.length).toEqual(2);
+    });
```

Comment:
I used this test and thought it was more informative as it compares the titles instead of just length:

```ts
    test('returns titles when index patterns are provided', () => {
      const indexPatternTitles = getIndexPatternTitles(mockIndexPatternSavedObjects);
      expect(indexPatternTitles).toEqual(['filebeat-*', 'auditbeat-*']);
    });
```